schema.sql

```diff
@@ -3,3 +3,12 @@
 CREATE TABLE animals (
     name varchar(100)
 );
+
+CREATE TABLE animals ( 
+    id SERIAL PRIMARY KEY, 
```

Comment:
- [ ] As per project requirements, the `id` field should be in `int` data type.
![image](https://github.com/ibugithub/Vet-Clinic/assets/62849872/fb85786a-4493-43a2-81d0-c45d8454b8bf)

- [OPTIONAL ] You could also make the id column not null and autoincremented.
`    id int NOT NULL IDENTITY(1,1)
`